deal_thief/__init__.py

```diff
@@ -0,0 +1,16 @@
+from pyramid.config import Configurator
+import os
+
+
+def main(global_config, **settings):
+    """ This function returns a Pyramid WSGI application.
+    """
+    settings['sqlalchemy.url'] = os.environ['DATABASE_URL']
+    config = Configurator(settings=settings)
+    config.include('pyramid_jinja2')
+    config.add_jinja2_renderer('.html')
```

Comment:
I like that you figured out you could control which file extensions would be counted as jinja2 files.  Good work.